resources/resolver.go

```diff
@@ -138,8 +170,9 @@ func Fetch(uri string, rsrc string) (io.ReadCloser, error) {
 // Size
 // Given a base URI and a resource name, determine the size of the resource.
 func Size(uri string, rsrc string) (uint, error) {
+	token := os.Getenv("HF_API_TOKEN")
```

Comment:
Maybe instead of repeating the `HF_API_TOKEN` check in four places, we should do it at initialization. DRY things up a little.

Either do an `init()` function that sets a module-level variable from the env, or do it at initialization of `main()`.